### PR TITLE
[ko] Translate the glossary term 'Ephemeral Container' into Korean

### LINKED ISSUE #35258: [ko] Translate the glossary term 'Ephemeral Container' into Korean
**This is a Feature Request**

**What would you like to be added**

Translate the glossary term 'Ephemeral Container' into Korean
- link to `en` entry: https://kubernetes.io/docs/reference/glossary/?fundamental=true#term-ephemeral-container
- link to `ko` entry: https://kubernetes.io/ko/docs/reference/glossary/?fundamental=true#term-ephemeral-container

**Why is this needed**
Not translated yet

**Comments**

/language ko
/assign

### PR DESCRIPTION
**Pull request by jihoon-seo**
Resolves #35258

### CODE REVIEW

**Review by sftim — COMMENTED** [review]
It's IMO useful to Korean readers to know that the term that the API uses is “ephemeral container” (actually `ephemeralContainers`, in the Pod API).

**Comment by sftim** [comment]
We could link to https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#EphemeralContainer even though the English doc doesn't mention this, provided that the justification for this link is helping readers understand what 임시 컨테이너 equates to in the API.

**Comment by seokho-son** [comment]
> It's IMO useful to Korean readers to know that the term that the API uses is “ephemeral container” (actually ephemeralContainers, in the Pod API).

Thank you @sftim :)
I have the same opinion with you. It will be useful to Korean readers.
So, the Korean l10n team has a convention in localizing the Glossary terms as well.
In case of glossary, we provides the original English term beside of the Korean title.

For instance,

https://raw.githubusercontent.com/kubernetes/website/main/content/ko/docs/reference/glossary/daemonset.md

```
title: 데몬셋(DaemonSet)
id: daemonset
date: 2018-04-12
full_link: /ko/docs/concepts/workloads/controllers/daemonset
short_description: >
  파드의 복제본을 클러스터 노드 집합에서 동작하게 한다.
```

@jihoon-seo could you apply the convention accordingly ?

**Comment by jihoon-seo** [comment]
@sftim @seokho-son Thank you for your comments!
I reflected your suggestions. PTAL!

File: content/ko/docs/reference/glossary/ephemeral-container.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+- fundamental
+---
+{{< glossary_tooltip text="파드" term_id="pod" >}} 내에 임시적으로 실행할 수 있는 {{< glossary_tooltip text="컨테이너" term_id="container" >}} 타입.
+
+<!--more-->
+
+문제가 있는 실행 중 파드를 조사하고 싶다면, 파드에 임시 컨테이너를 추가하고 진단을 수행할 수 있다. 임시 컨테이너는 리소스 및 스케줄링에 대한 보장이 제공되지 않으며, 워크로드 자체를 실행하기 위해 임시 컨테이너를 사용해서는 안 된다.
+
+<!-- Even though the English doc doesn't mention this, the link below is to help Korean readers understand what 임시 컨테이너 equates to in the API. -->
+더 자세한 정보는 파드 API의 [EphemeralContainer](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#EphemeralContainer)를 참고한다.
```
Comment (seokho-son) [inline content/ko/docs/reference/glossary/ephemeral-container.md:20]: @jihoon-seo 
해당 내용에 대해서 설명을 부탁드려도 될까요?
아시다시피 한글화의 경우 보통은 원문의 형태를 최대한 유지해왔었습니다. :)

Comment (jihoon-seo) [reply content/ko/docs/reference/glossary/ephemeral-container.md:20]: 무슨 이유에서인지 이 댓글에 대한 노티 메일을 못 받았네요.. 🥲

**Pull request merged**